CONTRIBUTING.md

```diff
@@ -1,6 +1,65 @@
-## Contributing
+# Contributing
 
-For now `master` is the main branch and code improvements are made in topic branches that get merged into it. Eventually there will be specific release branches, but for now `master` is the release branch.
+Planetary is an open-source project, as such, we openly welcome contributions of any sort: code improvement, bug fixes, translations, new features, bug reports...
+
+We encourage you to read this guide first or contact any of us.
+
+## Architecture
+
+**A good place to start: all of our important architectural decisions are being annotated inside the [Architecture folder](Architecture/) using [Architecture Decision Records](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions).**
+
+Planetary is organized around [major components](https://developer.apple.com/documentation/swift_packages/organizing_your_code_with_local_packages). A quick look to the dependency graph:
+
+```mermaid
+graph TD
+  Planetary((Planetary))
+  Planetary-->Logger
```

Comment:
Looking at the way the mermaid diagram is rendered, I think it would make more sense to me if the arrows were reversed? Maybe because I think of the imports as a tree with Planetary at the root, but this makes it seem like Logger is the root node. Does your brain see it the opposite way?

Reply:
The way I read A->B is A uses (or depends on) B, just like [the UML dependency relationship](https://www.uml-diagrams.org/dependency.html)

Reply:
Ok that makes sense.